src/main/groovy/io/jenkins/plugins/sentry/SentryPlugin.groovy

```diff
@@ -1,68 +0,0 @@
-package io.jenkins.plugins.sentry
-
-import hudson.Plugin
-import java.util.logging.*
-
-import io.sentry.Sentry
-import io.sentry.jul.SentryHandler
-
-class SentryPlugin extends Plugin {
-    Logger LOG = Logger.getLogger(SentryPlugin.name)
-
-    void postInitialize() {
-        if (!System.env.get('SENTRY_DSN')) {
-            LOG.error('The `SENTRY_DSN` environment variable is not defined, not configuring Sentry')
-            return
-        }
-
-        Sentry.init()
-
-        System.env.each { key, value ->
-            key.eachMatch('SENTRY_TAG_(\\w+)') { match ->
```

Comment:
is is not needed?

Reply:
Yes, it is done by SDK in modern versions